src/main/resources/spring/camel-context.xml

```diff
@@ -88,14 +118,16 @@
         <rest path="/mappings">
             <description>Mapping REST API</description>
             <get uri="/flag-assessment">
+                <param name="offset" type="query" defaultValue="0" required="false" description="Define the page requested" />
+                <param name="limit" type="query" defaultValue="{{pagination.flag-assessment.limit.default}}" required="false" description="Define the size of the requested page" />
```

Comment:
Same as above. I think these changes should be covered in https://github.com/project-xavier/xavier-integration/pull/114